tests/node/test_wakunode_store.nim

```diff
@@ -546,81 +567,270 @@ suite "Waku Store - End to End - Unsorted Archive":
   asyncTeardown:
     waitFor allFutures(client.stop(), server.stop())
 
-  asyncTest "Basic (Timestamp and Digest) Sorting Validation":
+  asyncTest "Basic (Timestamp and Hash) Sorting Validation":
     # When making a history query
-    let queryResponse = await client.query(historyQuery, serverRemotePeerInfo)
+    let queryResponse = await client.query(storeQuery, serverRemotePeerInfo)
 
-    # Then the response contains the messages
+    # Check the ordering
     check:
-      queryResponse.get().messages ==
-        @[
-          unsortedArchiveMessages[2],
-          unsortedArchiveMessages[0],
-          unsortedArchiveMessages[1],
-          unsortedArchiveMessages[4],
-          unsortedArchiveMessages[3],
-        ]
+      queryResponse.get().messages.len == 5
```

Comment:
We do check the ordering L578-597 but each run the messages have different timestamp and hashes and since the messages returned are ordered by hash then the order is different.

We could hard code the messages timestamp but I feel it would make the test less useful.

Reply:
> We do check the ordering L578-597 but each run the messages have different timestamp and hashes and since the messages returned are ordered by hash then the order is different.
> 
> We could hard code the messages timestamp but I feel it would make the test less useful.

Okay I see,
Aside from the test comment itself, something important is that the following should still work: https://github.com/waku-org/nwaku/issues/2317
( cc @richard-ramos )